.travis.yml

```diff
@@ -28,7 +28,7 @@ matrix:
       - env: DJANGO=master
 
 install:
-    - pip install tox tox-venv tox-travis
+    - pip install tox==3.7.0 tox-venv tox-travis
```

Comment:
New tox 3.8.0 causes problems in Travis (it was released just 20 hours ago, hopefully issue will be fixed soon)